[2022-08-19 02:44:47,679] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Satellite_Data.load_data manual__2022-08-19T02:44:40.366905+00:00 [queued]>
[2022-08-19 02:44:47,689] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Satellite_Data.load_data manual__2022-08-19T02:44:40.366905+00:00 [queued]>
[2022-08-19 02:44:47,689] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 02:44:47,689] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-19 02:44:47,689] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 02:44:47,709] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): load_data> on 2022-08-19 02:44:40.366905+00:00
[2022-08-19 02:44:47,715] {standard_task_runner.py:52} INFO - Started process 1698 to run task
[2022-08-19 02:44:47,719] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Satellite_Data', 'load_data', 'manual__2022-08-19T02:44:40.366905+00:00', '--job-id', '111', '--raw', '--subdir', 'DAGS_FOLDER/data_loader.py', '--cfg-path', '/tmp/tmp36jdinpq', '--error-file', '/tmp/tmpqb5uvuqi']
[2022-08-19 02:44:47,720] {standard_task_runner.py:80} INFO - Job 111: Subtask load_data
[2022-08-19 02:44:47,809] {task_command.py:371} INFO - Running <TaskInstance: Satellite_Data.load_data manual__2022-08-19T02:44:40.366905+00:00 [running]> on host a5d93a99773b
[2022-08-19 02:44:47,934] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Satellite_Data
AIRFLOW_CTX_TASK_ID=load_data
AIRFLOW_CTX_EXECUTION_DATE=2022-08-19T02:44:40.366905+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-19T02:44:40.366905+00:00
[2022-08-19 02:44:54,548] {python.py:173} INFO - Done. Returned value was: None
[2022-08-19 02:44:54,572] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=Satellite_Data, task_id=load_data, execution_date=20220819T024440, start_date=20220819T024447, end_date=20220819T024454
[2022-08-19 02:44:54,641] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-19 02:44:54,692] {local_task_job.py:273} INFO - 2 downstream tasks scheduled from follow-on schedule check
